lib/Checkout/Accounts/AccountsAccountHolder.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Checkout\Accounts;
+
+use Checkout\Common\AccountHolderIdentification;
+use Checkout\Common\Address;
+
+abstract class AccountsAccountHolder
```

Comment:
I think that AccountsAccount... is redundant. Maybe we can just name it AccountHolder? When we import this class, the path is going to be Checkout\Accounts\AccountHolder already